### ISSUE TITLE
 Found problem(s) parsing "Entity". See above?

### ISSUE THREAD

**Issue by followztx** [greenrobot/greenDAO#451]
Error:Execution failed for task ':app:greendao'.
> Found problem(s) parsing "/Users/followztx/work/studio_wp/galaxy4Git/app/src/main/java/com/onepointfive/galaxy/http/json/bookshelf/BsBookJson.java". See above

**but there no more info for why can't parse BsBookJson.java.**

```java
package com.onepointfive.galaxy.http.json.bookshelf;

import android.text.TextUtils;

import com.onepointfive.galaxy.http.json.JsonTag;

import org.greenrobot.greendao.annotation.Entity;
import org.greenrobot.greendao.annotation.Id;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by followztx on 16/7/22.
 */
@Entity
public class BsBookJson implements JsonTag {

    @Id
    public String BookId;
    public String SaltId;
    public String BookName;
    public int CreateTime;
    public int Status;
    public int LastModifyTime;
    public int LastPublishTime;
    public int PartnerShip;
    public int BookClassId0;
    public int BookClassId;
    public String BookClassName0;
    public String BookClassName;
    public int TotalChapters;
    public int TotalWords;
    public String NoteForMobile;
    public int HasVipChapter;
    public int SeriesStatus;
    public String CoverUrlM;
    public String ShareUrl;
    public int TotalPV;
    public int FavoriteNum;
    public int CommentNum;
    public int RewardAmount;
    public int ComedyNum;
    public int RecommendTicketNum;
    public int ShareNum;
    public int RewardNum;
    public int ReaderNum;
    public int UserId;
    public String NickName;
    public int Sex;
    public int IsVip;
    public int Level;
    public String AvatarUrlM;
    public int LastChapterId;
    public int UpdateTime;
    public int ReadNum;
    public int ReadWords;
    public int ReadProgress;
    public int ChapterNo;
    public int IsNew;


    //add for ui
    public boolean selected;
    public boolean isDownlaod;

    public static String getBookIds(List<BsBookJson> bsBookJsons){
        List<String> ids=new ArrayList<>();
        for(BsBookJson bsBookJson:bsBookJsons){
            ids.add(bsBookJson.BookId);
        }
        return TextUtils.join(",",ids);
    }
}
```

**Comment by sosite** [comment]
When you remove extra code like `getBookIds()` then it should works. I have the same issue so I back to `3.1.0` version.

**Comment by greenrobot-team** [comment]
Is this the same problem as #433? Check the log above the error statement, you might have to switch to the Gradle Console to see the full output. -ut

**Comment by sosite** [comment]
Yea, it seems the same issue - just with other error message.

**Comment by followztx** [comment]
add --debug  parameter,find  "TextUtils cannot be resolved"
`09:53:58.997 [ERROR] [system.err] Found 1 problem(s) parsing "/Users/followztx/work/studio_wp/galaxy4Git/app/src/main/java/com/onepointfive/galaxy/http/json/bookshelf/BsBookJson.java":
09:53:58.997 [ERROR] [system.err] Pb(50) TextUtils cannot be resolved`

**Comment by followztx** [comment]
so,i back all version to **3.1.0**

**Comment by greenrobot-team** [comment]
Alright, so this is the same issue as #433. Closing then.

This should now be fixed with the `3.2.0` release. It is already on Maven Central, still have to update the docs. -ut
